### DISCUSSION TITLE
New Org, New Communication

### DISCUSSION THREAD

**Discussion by Ry-E** [hopeYouLikeGamminToo/ourGame-Home#1] [Announcements]
## 👋 !
This kinda screws up the issues and wiki we had going but i think it might be worth it to merge everything over here. I think this will be a better way to discuss stuff other than with issues.

**Comment by howl0893** [comment]
Do we get notified when someone messages here? I didn't see this until now!
Side note: I've been out of town for the past two weeks for work fellers.. haven't made any progress :/

**Comment by bhowl** [comment]
This was sent to my email and im replying through outlook

**Comment by bhowl** [comment]
![image](https://user-images.githubusercontent.com/67282700/170115338-bf762b74-20cc-4f04-9965-a2d3df1561ad.png)

Go to the repo within the org and make sure you are a contributor matt, also could click 'watch' to get notification i think.

**Reply by bhowl** [reply]
I think you will be added as a contributor once you make a commit tho so maybe just hit the watch button

**Comment by bhowl** [comment]
![image](https://user-images.githubusercontent.com/67282700/170115570-b21cd83f-8c7c-44b3-b411-3e19493e9d9b.png)

**Comment by Ry-E** [comment]
Ive got a pixi app running on a server, but not really the way id like it to be running lol. I could upload it anyway and we could try get some new discussion next week
